README.md

```diff
@@ -1,14 +1,28 @@
 # Nautobot Lab
 
-`Nautobot Lab` is an all-in-one docker container that allows a user to quickly get an instance of nautobot up and running with minimal effort, for the purposes of kicking the tires.
+`Nautobot Lab` is an all-in-one docker container that allows a user to quickly get an instance of Nautobot up and running with minimal effort, for the purposes of kicking the tires. With that said; `nautobot-lab` is **NOT** a ready for production container image.
+
+## Running from Docker Hub
+Building the container isn't needed to get up and running quickly. The image is hosted on Docker Hub for public consumption. Since the image is an all-in-on container, it will take a a few seconds for all of the services to start and stabalize. Once the container has started and all services have stabalized, the web interface can be accessed via `http://localhost:8000`.
```

Comment:
```suggestion
Building the container yourself isn't needed to get up and running quickly. The image is hosted on Docker Hub for public consumption, and you can download and start it with a single command. Since the image is an all-in-one container, it will take a few seconds to download the container, then a few seconds more for all of the services to start and stabilize. Once the container has started and all services have stabilized, the web interface can be accessed via `http://localhost:8000`.
```